rnd/autogpt_server/autogpt_server/blocks/create_medium_post.py

```diff
@@ -138,6 +138,13 @@ def create_post(
         return response.json()
 
     def run(self, input_data: Input) -> BlockOutput:
+        missing_fields = [field for field in [
+            'title', 'content', 'content_format', 'tags', 'publish_status'
+        ] if not getattr(input_data, field)]
+        if missing_fields:
+            yield "error", f"Missing required fields: {', '.join(missing_fields)}"
+            return
```

Comment:
The input is already validated by the graph and node executors, so there should be no need for this. Are you experiencing issues?
https://github.com/Significant-Gravitas/AutoGPT/blob/master/rnd/autogpt_server/autogpt_server/executor/manager.py